licenses/classifier.go

```diff
@@ -63,21 +62,25 @@ func (c *googleClassifier) Identify(licensePath string) ([]License, error) {
 		return nil, err
 	}
 
+	foundLicenseNames := map[string]struct{}{}
+
 	licenses := []License{}
 	for _, match := range matches.Matches {
 		if match.MatchType != "License" {
 			continue
 		}
 
+		// Skip duplicate licenses.
```

Comment:
This is an unrelated improvement: if the same license is found in a LICENSE file twice, only display it once.